src/pylero/test_run.py

```diff
@@ -446,7 +446,8 @@ def search(cls, query, fields=["test_run_id"], sort="test_run_id",
         if p_fields:
             function_name += "WithFieldsLimited"
             parms.append(p_fields)
-        parms.append(limit)
```

Comment:
- just a tab here is enough and no need for `if` check
- it seems `suds-py3` was dropping extra args but `suds-community` is supplying every arg even if the WSDL API doesn't accept
- not sure which one is good though 😅, auto-correcting user incorrect inputs or making user to strictly adhere to the API